test/sql/function/numeric/test_pg_math.test

```diff
@@ -97,3 +97,7 @@ select sign(3)
 ----
 1
 
+query I
+select log(2, 64)
```

Comment:
Can we add tests with a logarithm of 0, negative numbers, NaN and inf?

Reply:
added some spicy log corner case tests. NaN/Inf still ToDO

Reply:
also added NaN and Inf, results consistent with Python